### PR TITLE
React native widget data

### PR DESCRIPTION
**Pull request by PetrosZip**

### CODE REVIEW

File: QuadPaySDK/PaymentWidget/PaymentWidget.swift [hunk 1]
```diff
@@ -107,8 +107,11 @@ extension PaymentWidget {
         stackView.addArrangedSubview(paymentWidgetHeaderText)
         stackView.addArrangedSubview(paymentWidgetSubText)
         stackView.addArrangedSubview(timelapseGraphView)
+        stackView.addArrangedSubview(feeTierView)
         if(maxFee != 0){
-            stackView.addArrangedSubview(feeTierView)
+            feeTierView.isHidden = false
+        }else{
+            feeTierView.isHidden = true
```
Comment (rhyswilliamszip) [inline QuadPaySDK/PaymentWidget/PaymentWidget.swift:114]: Can this not be changed to 

`feeTierView.isHidden = maxFee != 0`

File: QuadPaySDK/PaymentWidget/PaymentWidget.swift [hunk 2]
```diff
@@ -159,20 +163,21 @@ extension PaymentWidget {
...
                     
                 case .failure(let error):
                     print(error)
+                    self.maxFee = 0
                     DispatchQueue.main.async {
                         self.layout()
                     }
                     
                 }
             }
```
Comment (rhyswilliamszip) [inline QuadPaySDK/PaymentWidget/PaymentWidget.swift:178]: Should there be a return statement here?

File: QuadPaySDK/PaymentWidget/RNPaymentWidget.swift [hunk 3]
```diff
@@ -110,21 +96,84 @@ extension RNPaymentWidget {
...
         stackView.addArrangedSubview(paymentWidgetHeaderText)
         stackView.addArrangedSubview(paymentWidgetSubText)
         stackView.addArrangedSubview(timelapseGraphView)
-    
+        stackView.addArrangedSubview(feeTierView)
+        if(maxFee != 0){
+            feeTierView.isHidden = false
+        }else{
+            feeTierView.isHidden = true
+        }
```
Comment (rhyswilliamszip) [inline QuadPaySDK/PaymentWidget/RNPaymentWidget.swift:112]: Same as flagged before.

**Review by rhyswilliamszip — CHANGES_REQUESTED** [review]

File: QuadPaySDK/PaymentWidget/PaymentWidget.swift (hunk 1 above)
Comment (PetrosZip) [reply QuadPaySDK/PaymentWidget/PaymentWidget.swift:114]: I tested it and is not hiding the view.

File: QuadPaySDK/PaymentWidget/RNPaymentWidget.swift (hunk 3 above)
Comment (PetrosZip) [reply QuadPaySDK/PaymentWidget/RNPaymentWidget.swift:112]: Same as above.

File: QuadPaySDK/PaymentWidget/PaymentWidget.swift (hunk 2 above)
Comment (PetrosZip) [reply QuadPaySDK/PaymentWidget/PaymentWidget.swift:178]: I dont think return statement is actually needed as we want to proceed on the dispatchQueue to unblock the thread so adding a return before line 172 will cause issues.

**Review by rhyswilliamszip — APPROVED** [review]

**Pull request merged**